keyboards/lily58/r2g/r2g.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
This file can probably be removed. Will need to remove its inclusion elsewhere and whatnot of course.